From e964195c2b0a68a83f265f19e76a2de285bf5768 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Mon, 18 Apr 2022 20:36:39 +0200
Subject: [PATCH] hwdb: 60-keyboard: Add Acer Aspire One AO532h keymappings

Add keymappings for the Acer Aspire One AO532h netbook.

Unmap the brightnesskeys because they send duplicate key events with
the ACPI video bus key events and add a mapping for the bluetooth
on/off hotkey.

(cherry picked from commit 5e3b8aeef30f8b46669280ecffc178fa43c8bf00)

Related: #2087778
---
 hwdb.d/60-keyboard.hwdb | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index adf5e9ac35..2555b8d35a 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -154,6 +154,12 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnAspire*8930:*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnAspire*7750G:*
  KEYBOARD_KEY_e0=!pageup
 
+# Acer Aspire One AO532h
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnAO532h:*
+ KEYBOARD_KEY_84=bluetooth
+ KEYBOARD_KEY_ce=unknown  # Brightness Up, also emitted by acpi-video, ignore
+ KEYBOARD_KEY_ef=unknown  # Brightness Down, also emitted by acpi-video, ignore
+
 # Predator PH 315-52
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnPredator*PH*315-52:*
  KEYBOARD_KEY_ef=kbdillumup                             # Fn+F10
